src/marlin_stubs/M330.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+namespace PrusaGcodeSuite {
+
+/// Metric-related GCodes.
+/// TODO: This is WIP. Those Gcodes need redesign.
```

Comment:
As a proof of concept this is ok, however discussing this with teams developing Marlin, Repetier and others is necessary, ideally into FW 4.3 final